source/whats-new.txt

```diff
@@ -12,12 +12,25 @@ What's New
 
 Learn what's new in:
 
+* :ref:`Version 1.5 <mongodb-analyser-whats-new-1.5>`
 * :ref:`Version 1.4 <mongodb-analyzer-whats-new-1.4>`
 * :ref:`Version 1.3 <mongodb-analyzer-whats-new-1.3>`
 * :ref:`Version 1.2 <mongodb-analyzer-whats-new-1.2>`
 * :ref:`Version 1.1 <mongodb-analyzer-whats-new-1.1>`
 * :ref:`Version 1.0 <mongodb-analyzer-whats-new-1.0>`
 
+.. _mongodb-analyzer-whats-new-1.5:
+
+What's New in 1.5
+-----------------
+
+New features of the 1.5 {+product+} release include:
+
+- New analysis rules introduced for EFCore to MQL, which enable displaying
+  MQL MongoDB Entity Framework Core Provider expressions at compile time
```

Comment:
MQL is no longer used in our public-facing documentation. We should use "MongoDB Query API" now:
```suggestion
- New analysis rules introduced for Entity Framework Core, which enable displaying
  MongoDB Query API translations for MongoDB Entity Framework Core Provider expressions at compile time
```